src/components/modal/CdrModal.vue

```diff
@@ -329,7 +321,7 @@ export default defineComponent({
         :class="mapClasses(style, 'cdr-modal__contentWrap', 'cdr-modal__dialog')"
         tabIndex="-1"
         :role="role"
-        :aria-modal="!!opened"
+        aria-modal="true"
```

Comment:
This should always be set to true. It was previously, because it was being statically set in the `dialogAttrs` and overwriting this dynamic binding. This change just cleans that up